app/src/marketplace/containers/ProductController/ValidationContextProvider.jsx

```diff
@@ -125,6 +126,18 @@ function useValidationContext(): ContextProps {
             clearStatus('streams')
         }
 
+        if (product.termsOfUse != null && product.termsOfUse.termsUrl) {
+            const validator = yup.string()
+                .trim()
+                .url()
+            const result = validator.isValidSync(product.termsOfUse.termsUrl)
+            if (!result) {
+                setStatus('termsOfUse', ERROR, 'Invalid URL for detailed terms')
```

Comment:
Untranslated string here

Reply:
Oh, actually the whole file is untranslated. I'm leaving this untranslated for now. It's more convenient to add localization in one go.

Reply:
ok, let's add an issue about it that we won't forget 👍

Reply:
I'll add 👍